### PR TITLE
BaSD-MS-68: sessions entity- CRUD

### PR DESCRIPTION
**Pull request by Micaela-89**
**Sessions Screens**

**List screen allows to:**
- Visualize all sessions
- Delete sessions
- Click on any to update it
- Click to add a new session

**Form screen allows to:**
 - Create a new session 
 - Edit an existing session
 This will be updated in the list screen.

### CODE REVIEW

File: src/Components/Sessions/Form/form.module.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  padding: 0px 40px 40px 40px;
+}
+.form-subscription li {
+  padding: 10px;
+}
+.form {
+  display: flex;
+  flex-direction: column;
+}
+label {
```
Comment (estefrare) [inline src/Components/Sessions/Form/form.module.css]: do not use pure tags

File: .eslintrc.json [hunk 2]
```diff
@@ -1,34 +1,33 @@
...
-        "react/react-in-jsx-scope": "off",
-        "prettier/prettier": [
-            "error",
-            {
-              "endOfLine": "auto"
-            }
-          ],
-        "react/prop-types": 0
-    }
+	"settings": {
```
Comment (estefrare) [inline .eslintrc.json]: I don't know what you changed here, but discard those changes

File: src/Components/Sessions/Modal/modal.module.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
+.container{
+  position:absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 120px;
+  width: 100%;
+  background-color: rgba(247, 233, 233, 0.5);
+}
```
Comment (estefrare) [inline src/Components/Sessions/Modal/modal.module.css:9]: please, just use the styles I am using here https://github.com/BaSD-2021/Esteban-MindSet-app/pull/3/files#diff-ed523f7f05eb0244fb2951825b943d3c5a120bad0e90f877acee0683ff0ca377R1-R9

**Review by estefrare — CHANGES_REQUESTED** [review]

File: src/Components/Sessions/Form/form.module.css (hunk 1 above)
Comment (Micaela-89) [reply src/Components/Sessions/Form/form.module.css]: Already fixed

File: .eslintrc.json (hunk 2 above)
Comment (Micaela-89) [reply .eslintrc.json]: Done

File: src/Components/Sessions/Modal/modal.module.css (hunk 3 above)
Comment (Micaela-89) [reply src/Components/Sessions/Modal/modal.module.css:9]: Styles changed

**Review by estefrare — APPROVED** [review]

**Review by nailaghigo — APPROVED** [review]
looks good to me !

**Pull request merged**